src/components/apiComponents/exampleSelector.jsx

```diff
@@ -0,0 +1,57 @@
+import { Select } from '@headlessui/react'
+import { ChevronDownIcon } from '@heroicons/react/20/solid'
+import clsx from 'clsx'
+
+const ApiExampleSelector = ({
+  examples,
+  selectedExample,
+  handleSetExample,
+}) => {
+  return (
+    <div className="w-full">
+      <label
+        className="mb-3 text-sm font-medium text-gray-800 dark:text-neutral-400"
+        htmlFor="endPoint"
+      >
+        Loaded Example
```

Comment:
Should this be called like "Preloaded Example"?  Not sure about exact terminology.

Reply:
I didn't use the term 'pre' because I load nothing on page load at the moment. So the Loaded Example is nothing, nothing is load.

Do we want to pre load an example into active state?